docker-storage-setup.sh

```diff
@@ -346,11 +346,58 @@ create_lvm_thin_pool () {
   lvconvert -y --zero n $CHUNK_SIZE_ARG --thinpool $VG/$DATA_LV_NAME --poolmetadata $VG/$META_LV_NAME
 }
 
+get_configured_thin_pool() {
+  local options tpool opt
+
+  if [ ! -f "$DOCKER_STORAGE" ];then
+    return 0
+  fi
+
+  if ! options=$(get_docker_storage_options); then
+    return 1
+  fi
+
+  # DOCKER_STORAGE_OPTIONS= is empty there is no storage driver configured yet.
+  [ -z "$options" ] && return 0
+
+  # No thin pool configured
+  if ! grep -q -e "--storage-opt dm.thinpooldev=" <<< $options;then
```

Comment:
Looks like this code is currently only called if `[ "$STORAGE_DRIVER" == "devicemapper" ]` so maybe just drop this hunk?